src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -120,7 +168,14 @@ class DayPickerKeyboardShortcuts extends React.Component {
           )}
           type="button"
           aria-label={toggleButtonText}
-          onClick={this.onClick}
+          onClick={this.onShowKeyboardShortcutsButtonClick}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') {
+              e.preventDefault();
+            } else if (e.key === 'Space') {
+              this.onShowKeyboardShortcutsButtonClick();
```

Comment:
this should probably pass the `e` through?

Reply:
i think it'd be more future-proof if it ever needed it in the future. Not a huge deal obv.